stdlib/tarfile.pyi

```diff
@@ -121,16 +119,124 @@ def open(
     pax_headers: Mapping[str, str] | None = ...,
     debug: int | None = ...,
     errorlevel: int | None = ...,
-    compresslevel: int | None = ...,
```

Comment:
I see this argument being used in the implementation, why did you remove it?

Reply:
Due to this comment from the docs:

> For modes 'w:gz', 'x:gz', 'w|gz', 'w:bz2', 'x:bz2', 'w|bz2', [tarfile.open()](https://docs.python.org/3/library/tarfile.html#tarfile.open) accepts the keyword argument compresslevel (default 9) to specify the compression level of the file.